pytorch_lightning/loggers/tensorboard.py

```diff
@@ -286,6 +287,12 @@ def _get_next_version(self):
 
         return max(existing_versions) + 1
 
+    @staticmethod
+    def _sanitize_params(params: Dict[str, Any]) -> Dict[str, Any]:
+        params = LightningLoggerBase._sanitize_params(params)
```

Comment:
For later, should we move this out as a utility function? Are there opportunities for consolidation with other hyper parameter saving logic the framework has?

Reply:
We can but keep in mind not all loggers have the same level of support and flexibility so sanitization, flattening etc. is generally speaking always subject to the individual logger.

> Are there opportunities for consolidation with other hyper parameter saving logic the framework has?

not any I know of